release/release_tests.yaml

```diff
@@ -3823,22 +3823,11 @@
   cluster:
     byod:
       type: gpu
-    cluster_compute: single_node_benchmark_compute.yaml
+    cluster_compute: multi_node_autoscaling_compute.yaml
 
   run:
-    # Expect the test to finish around 40 seconds.
-    timeout: 100
-    script: python parquet_metadata_resolution.py --num-files 915 --cloud aws
-
-  variations:
-    - __suffix__: aws
-    - __suffix__: gce
-      env: gce
-      frequency: manual
-      cluster:
-        cluster_compute: single_node_benchmark_compute_gce.yaml
-      run:
-        script: python parquet_metadata_resolution.py --num-files 915 --cloud gcp
+    timeout: 500
```

Comment:
Will update this accordingly when https://buildkite.com/ray-project/release/builds/26586# finishes